lldb/test/API/functionalities/progress_reporting/swift_progress_reporting/main.swift

```diff
@@ -0,0 +1,49 @@
+struct IntPair {
```

Comment:
Was this copied from another test? If so, can we use the original source file instead of duplicating it? It doesn't seem like the content of this tests is particularly relevant for the test. I think it's fine to have a path that goes outside the current directory in the Makefile.

Reply:
This was copied from the SwiftUnsafe data formatter test (that I wrote few years ago). I chose it because it creates various types of variables, that will - hopefully - exercise the type-system to report events. However, I modified it a bit because there were some bits that are relevant to this test (i.e. turning an array to a `UnsafePointer / UnsafeBufferPointer etc.).

@adrian-prantl may be you have some suggestions for a simpler test ?

Reply:
You'll want to use a test that imports other modules. The test I added to https://github.com/apple/llvm-project/pull/4083/files has a pretty minimal example for adding a framework (you can remove the hidden second framework and you also don't need to use a .in file for the source code of the dylib).

Reply:
@adrian-prantl I updated the test in https://github.com/apple/llvm-project/pull/4085/commits/d1bfff86e80dafd512c7bd2dd49ee79a4851bc28